gpkitmodels/GP/aircraft/motor/motor.py

```diff
@@ -41,7 +41,7 @@ class Motor(Model):
 
     Variables
     ---------
-    Qstar       .8          [kg/(N*m)]         motor specific torque
+    Qstar       .5          [kg/(N*m)]         motor specific torque
```

Comment:
.5 is pretty sporty based on the numbers from the STOL guys; .8 is more in line with their model at representative powers

Reply:
It wasn't solving for higher values.  I can try again.  Also not convinced that the STOL guys numbers are that accurate...

Reply:
Yea it was a fit of that one manufacturer so you may be able to get lighter; it's a high-sensitivity variable for sure.